.circleci/test-deploy.yml

```diff
@@ -278,6 +348,11 @@ workflows:
             parameters:
               executor: [linux, windows]
 
+      - integration-test-run-build:
+          matrix:
+            parameters:
+              executor: [linux, windows]
```

Comment:
I think we can include macOS in these now: https://discuss.circleci.com/t/announcing-macos-support-for-free-plans-is-here/42826